src/components/security/security-controller.ts

```diff
@@ -59,11 +59,21 @@ export async function securityGet(req: Request, res: Response): Promise<void> {
       })
     : [];
 
+  const hasAuthAppMethod = req.session.mfaMethods?.some((method) => {
+    return method.mfaMethodType === "AUTH_APP";
+  });
```

Comment:
Do we need this? I added this in, but based on the wording of the message, I think we would always want to show the message when the user only has one method

Reply:
@saralk , do you still need an answer to this?

Reply:
https://gds.slack.com/archives/C02F70D2LQN/p1712149445312499